src/Mod/CAM/Path/Machine/ui/editor/machine_editor.py

```diff
@@ -0,0 +1,1478 @@
+            widget = self.axes_layout.itemAt(i).widget()
+            if widget:
+                widget.setParent(None)
+
+        # Get current type
+        type_key = self.type_combo.itemData(self.type_combo.currentIndex())
+        if not type_key:
+            return
+        config = self.MACHINE_TYPES[type_key]
+
+        # Create axes group
+        all_axes = config.get("linear", []) + config.get("rotary", [])
+        if not all_axes:
+            self.axes_group.setVisible(False)
+            return
+
+        axes_form = QtGui.QFormLayout()
+
+        # Get axes directly from machine object
+        linear_axes = list(self.machine.linear_axes.keys()) if self.machine else []
+        rotary_axes = list(self.machine.rotary_axes.keys()) if self.machine else []
+
+        # Linear axes
+        if linear_axes:
+            linear_group = QtGui.QGroupBox("Linear Axes")
+            linear_layout = QtGui.QFormLayout(linear_group)
+
+            for axis in linear_axes:
+                axis_obj = self.machine.linear_axes[axis]
+                converted_min = (
+                    axis_obj.min_limit if units == "metric" else axis_obj.min_limit / 25.4
+                )
+                min_edit = QtGui.QLineEdit()
+                min_edit.setText(f"{converted_min:.2f}{length_suffix}")
+                min_edit.editingFinished.connect(
+                    lambda edit=min_edit, suffix=length_suffix.strip(), ax=axis, fld="min": self.normalize_text(
+                        edit, suffix, ax, fld
+                    )
+                )
+
+                converted_max = (
+                    axis_obj.max_limit if units == "metric" else axis_obj.max_limit / 25.4
+                )
+                max_edit = QtGui.QLineEdit()
+                max_edit.setText(f"{converted_max:.2f}{length_suffix}")
+                max_edit.editingFinished.connect(
+                    lambda edit=max_edit, suffix=length_suffix.strip(), ax=axis, fld="max": self.normalize_text(
+                        edit, suffix, ax, fld
+                    )
+                )
+
+                converted_vel = (
+                    axis_obj.max_velocity if units == "metric" else axis_obj.max_velocity / 25.4
+                )
+                vel_edit = QtGui.QLineEdit()
+                vel_edit.setText(f"{converted_vel:.2f}{vel_suffix}")
+                vel_edit.editingFinished.connect(
+                    lambda edit=vel_edit, suffix=vel_suffix.strip(), ax=axis, fld="max_velocity": self.normalize_text(
+                        edit, suffix, ax, fld
+                    )
+                )
+
+                axis_layout = QtGui.QHBoxLayout()
+                axis_layout.addWidget(QtGui.QLabel("Min:"))
+                axis_layout.addWidget(min_edit)
+                axis_layout.addWidget(QtGui.QLabel("Max:"))
+                axis_layout.addWidget(max_edit)
+                axis_layout.addWidget(QtGui.QLabel("Max Vel:"))
```

Comment:
```suggestion
                axis_layout.addWidget(QtGui.QLabel("Max Vel"))
```

Is this translatable? wouldn't it be better to spell it out so translators would know what this means (they just see "Max Vel" as new string.